pkg/ddc/jindo/operations/cached.go

```diff
@@ -16,11 +17,15 @@ func (a JindoFileUtils) CleanCache() (err error) {
 
 	stdout, stderr, err = a.exec(command, false)
 
-	time.Sleep(30 * time.Second)
-
 	if err != nil {
 		err = fmt.Errorf("execute command %v with expectedErr: %v stdout %s and stderr %s", command, err, stdout, stderr)
+		if utils.IgnoreNotFound(err) == nil {
+			fmt.Printf("Failed to clean cache due to %v", err)
+			return nil
+		}
 		return
+	} else {
+		time.Sleep(30 * time.Second)
```

Comment:
Why do you add sleep 30s?

Reply:
Currently, clean cache is an asynchronous task under jindoruntime, so you need to sleep for a period of time after the command is executed, and this task can be finished by 30 seconds at most. will optimize and make it a synchronous task in the future